client/table.go

```diff
@@ -40,30 +40,43 @@ import (
 //   eventually wind up on the server using new table-based requests to perform
 //   operations.
 //
-// - Create tables and schemas. Lookup table schema in BindModel.
-//
-// - Use table ID in primary key.
-//
 // - Enhance DelRange to handle model types? Or add a DelStructRange?
 //
 // - Naming? PutStruct vs StructPut vs TablePut?
 //
 // - Need appropriate locking for the DB.experimentalModels map.
 //
-// - Normalize column names to lowercase.
-//
 // - Allow usage of `map[string]interface{}` in place of a struct. Probably
 //   need table schemas first so we know which columns exist.
 //
 // - Add support for namespaces. Currently namespace ID 0 is hardcoded.
 
+var (
+	// TODO(pmattis): Use the appropriate constant in keys.go when it appears.
+	tableDataPrefix = []byte("tbl-")
```

Comment:
Why do we need anything here? If the access to cockroach is limited to tables...